src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/ImportVolumeGraphicsFile.cpp

```diff
@@ -0,0 +1,69 @@
+#include "ImportVolumeGraphicsFile.hpp"
+
+#include "complex/DataStructure/DataArray.hpp"
+#include "complex/DataStructure/DataGroup.hpp"
+#include "complex/DataStructure/Geometry/ImageGeom.hpp"
+
+using namespace complex;
+namespace fs = std::filesystem;
+
+namespace
+{
+inline constexpr int32 k_VolBinaryAllocateMismatch = -91504;
+inline constexpr int32 k_VolOpenError = -91505;
+inline constexpr int32 k_VolReadError = -91506;
```

Comment:
if we are going to be using `usize` then we should be using `int32` instead of `int32_t`